diff --git a/package/nanokvm-sg200x/nanokvm-sg200x.mk b/package/nanokvm-sg200x/nanokvm-sg200x.mk
index ef1d0bd8cd..9705e80947 100644
--- a/package/nanokvm-sg200x/nanokvm-sg200x.mk
+++ b/package/nanokvm-sg200x/nanokvm-sg200x.mk
@@ -156,6 +156,9 @@ define NANOKVM_SG200X_INSTALL_TARGET_CMDS
 	if [ "X$(BR2_PACKAGE_TAILSCALE_RISCV64)" != "Xy" ]; then \
 		rm -f $(TARGET_DIR)/kvmapp/system/init.d/S??tailscaled ; \
 	fi
+
+	rm -f "$(TARGET_DIR)/kvmapp/system/init.d/S15kvmhwd copy"
+
 	mkdir -pv $(TARGET_DIR)/etc/init.d/
 	rsync -r --verbose --copy-dirlinks --copy-links --hard-links $(TARGET_DIR)/kvmapp/system/init.d/ $(TARGET_DIR)/etc/init.d/
 	if [ -e $(NANOKVM_SG200X_EXT_OVERLAY)/etc/init.d ]; then \
